### PR TITLE
ML-644: Improve Language Detection Model

### PR DESCRIPTION
**Pull request by rosequ**
### Why was this PR opened? ###
JIRA stories:
https://adasupport.atlassian.net/browse/ML-644

Some of the language profiles contain characters that belong to other languages since all the profiles were created from Wikipedia data by a script. This PR removes the non-language characters for languages that does not sue Latin script.

### What are the changes? ###
- added a script to clean the language profiles
- updated some of the language profiles

### How can someone test the changes? ###
```python
>>>from langdetect import detect_langs
>>>detect_langs("為何我於預訂時來回程都選擇$210的超值套餐，包括行李、餐點及選擇座位，但訂票時不能選擇座位，只由你們公司安排，如及後更改須額外付款，請問是甚麼原因?")
```
should now give you `zh-tw` as the output rather than `ko`, which used to be the case formerly.

---
- [x] PR description present and accurate
- [x] Code changes are documented
~- [ ] Unit tests cover code changes~
~- [ ] PR was tested on ada-dev~
- [x] PR title follows the format \<Jira Card\>: \<Change description\>
~- [ ] Deployment plan is included in PR if required~

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,6 +24,15 @@ Languages
     pt, ro, ru, sk, sl, so, sq, sv, sw, ta, te, th, tl, tr, uk, ur, vi, zh-cn, zh-tw
 
 
+Clean Language Profiles
+=======================
+Some of the language profiles contain characters that belong to other languages since all the profiles were created automatically. Run the following script to clean
+the language profiles:
+
+```bash
+python scripts/clean_language_profiles.py
```
Comment (gordania) [inline README.md:33]: I think the script didn't get checked in

Comment (rosequ) [reply README.md:33]: Sorry, just updated it!

**Review by rahelehmaki — APPROVED** [review]
Looks great 🎉

File: scripts/clean_language_profiles.py [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+import json
+
+
+def is_in_language(ngram, begin_unicode_range, end_unicode_range):
+    """
+    Checks if all the characters in the ngram are in the unicode range
+    of the language.
+    """
+    for character in ngram:
+        if not (begin_unicode_range <= ord(character) <= end_unicode_range):
```
Comment (gordania) [inline scripts/clean_language_profiles.py:15]: Cool, I didn't know about the `ord` function!

**Review by gordania — APPROVED** [review]
Nice!

**Review by salman1993 — APPROVED** [review]

**Pull request merged**